src/test/java/org/opensearch/geospatial/ip2geo/jobscheduler/DatasourceTests.java

```diff
@@ -0,0 +1,48 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+
+package org.opensearch.geospatial.ip2geo.jobscheduler;
+
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+import static org.opensearch.geospatial.plugin.GeospatialPlugin.IP2GEO_DATASOURCE_INDEX_NAME_PREFIX;
+
+import java.time.Instant;
+import java.util.ArrayList;
+import java.util.Locale;
+
+import org.opensearch.geospatial.ip2geo.common.DatasourceManifest;
+import org.opensearch.test.OpenSearchTestCase;
+
+public class DatasourceTests extends OpenSearchTestCase {
+    public void testCurrentIndexName() {
+        String id = "test";
```

Comment:
shall we create random string here?

Reply:
Please review on this PR. https://github.com/opensearch-project/geospatial/pull/257
Current PR will be ready once parent PR is closed.